ArtOfIllusion/src/artofillusion/ArtOfIllusion.java

```diff
@@ -272,7 +273,7 @@ public static void newWindow()
   }
 
   /** Create a new window for editing the specified scene. */
-
+  @SuppressWarnings("CallToPrintStackTrace")
```

Comment:
Another thing I was wondering about: @peastman, do we want to get into the habit of suppressing compiler warnings?